src/java.base/share/classes/java/security/spec/PSSParameterSpec.java

```diff
@@ -64,13 +64,18 @@
  *   ...  -- Allows for future expansion --
  * }
  * </pre>
- * <p>Note: the PSSParameterSpec.DEFAULT uses the following:
+ * <p>Note: the {@code PSSParameterSpec.DEFAULT} uses the following:
  *     message digest  -- "SHA-1"
  *     mask generation function (mgf) -- "MGF1"
  *     parameters for mgf -- MGF1ParameterSpec.SHA1
  *     SaltLength   -- 20
  *     TrailerField -- 1
  *
+ * <p>Its values are based on the default values defined in the ASN.1 encoding
+ * from the PKCS#1 standard and may become obsolete as time progresses.
+ * Please do not rely on {@code PSSParameterSpec.DEFAULT} unless these
+ * values are really what you want to use.
```

Comment:
Some suggested rewordings: 

`Its values are based on the default values defined in the PKCS#1 standard, as defined above. Some of these defaults are no longer recommended due to advances in cryptanalysis -- see RFC 8017 for more details. Thus, {@code PSSParameterSpec.DEFAULT} is deprecated and should be used at your own risk.`

Reply:
Thanks for the rewording suggestion. This part has been removed.